src/main/java/kostka/moviecatalog/service/communication/WebClientCommunicationServiceImpl.java

```diff
@@ -0,0 +1,36 @@
+package kostka.moviecatalog.service.communication;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+import org.springframework.web.reactive.function.BodyInserters;
+import org.springframework.web.reactive.function.client.WebClient;
+
+@Service("WebClientCommService")
```

Comment:
See the previous pull request. 
But if you want to use both services at the same time, this makes sense probably.